catalyst/dl/utils/formatters.py

```diff
@@ -44,12 +44,21 @@ class TxtMetricsFormatter(MetricsFormatter):
     def __init__(self):
         super().__init__("[{asctime}] ")
 
+    def _format_metric(self, name, value):
+        if name in {"_base/lr"}:
```

Comment:
what if we would print in scientific notation not only LR, but all the metrics that are below 1e-4?